wiki/Tournaments/BTT/5/en.md

```diff
@@ -29,9 +29,9 @@ The **Beginner's Taiko Tournament 5** (***BTT5***) was a worldwide double-elimin
 
 | Placing | Prizes |
 | :-: | :-- |
-| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 4 months of osu!supporter tag, 45€ |
-| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag, 25€ |
-| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag, 15€ |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 4 months of osu!supporter, 45€ |
```

Comment:
🔬  Another case of "unique profile badge" coming first (I realise this is not a critical issue)